projects/gateway2/translator/sslutils/ssl_utils.go

```diff
@@ -1,12 +1,30 @@
 package sslutils
 
 import (
+	"context"
 	"crypto/tls"
 	"fmt"
+	"strings"
 
+	"github.com/hashicorp/go-multierror"
 	"github.com/rotisserie/eris"
+	"github.com/solo-io/gloo/projects/gateway2/wellknown"
+	"github.com/solo-io/gloo/projects/gloo/pkg/api/v1/ssl"
+	"github.com/solo-io/go-utils/contextutils"
+	"google.golang.org/protobuf/types/known/wrapperspb"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/client-go/util/cert"
+	gwv1 "sigs.k8s.io/gateway-api/apis/v1"
+)
+
+const (
+	GatewaySslOptionsPrefix = wellknown.GatewayAnnotationPrefix + "/ssl"
+
+	GatewaySslCipherSuitesOption   = GatewaySslOptionsPrefix + "/cipher-suites"
```

Comment:
Super nit: why use `Option` suffix on this one and none of the others?
```suggestion
	GatewaySslCipherSuites   = GatewaySslOptionsPrefix + "/cipher-suites"
```